README.adoc

```diff
@@ -57,6 +57,8 @@ This is a standard Go package, and it installs like you might expect.
 To install it in your `$GOPATH` (assuming `$GOPATH/bin` is on your path), just use `go install`:
 
     $ go get -u github.com/bytesparadise/libasciidoc
+    $ cd $GOPATH/src/github.com/bytesparadise/libasciidoc
```

Comment:
I think we could replace the `go get` command with something like `git checkout github.com/bytesparadise/libasciidoc`. The `go get` is more for users who want to add the library to their project, whereas the `make install` is to have `libasciidoc` available as a command

Reply:
I've pushed a new commit that updates this - let me know if that's what you meant?